src/Integration.Vsix/Commands/PackageCommandId.cs

```diff
@@ -28,6 +28,7 @@ internal enum PackageCommandId
         ProjectTestPropertyAuto = 0x102,
         ProjectTestPropertyTrue = 0x103,
         ProjectTestPropertyFalse = 0x104,
+        SonarLintHelpShowLogs = 0x1025,
```

Comment:
Open question regarding the names of future commands: would we want to have the SonarLint prefix? Or would just their group suffice? i.e. HelpShowLogs, HelpViewDocumentation, HelpReportAnIssue, etc